docs/Glossary.md

```diff
@@ -230,7 +230,8 @@ A *route pattern* (or "path") is a string that describes a portion of a URL. Pat
 
   - `:paramName` – matches a URL segment up to the next `/`, `?`, or `#`. The matched string is called a [param](#params)
   - `()` – Wraps a portion of the URL that is optional
-  - `*` – Matches all characters (non-greedy) up to the next character in the pattern, or to the end of the URL if there is none, and creates a `splat` [param](#params)
+  - `*` – Matches all characters (non-greedy) up to the next character in the pattern, or to the end of the URL if there is none, and creates a `splat` [param](/docs/Glossary.md#params)
```

Comment:
I'll do that tonight. For now, I plan to update just the pattern-matching specific text.

I'll be happy to raise a separate PR to fix all other doc links.